components/TextFields/tests/snapshot/resources/TextFieldsSnapshotTests.xcassets/ic_done.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This asset appears unused, is there a reason for adding it?

Reply:
It was simpler to just copy-paste everything from the examples to the tests. We can make use of it later.